app/views/CannedResponsesListView/Dropdown/index.tsx

```diff
@@ -50,16 +49,15 @@ class Dropdown extends React.Component {
 	};
 
 	render() {
-		const { isMasterDetail, insets, theme, currentDepartment, onDepartmentSelected, departments } = this.props;
-		const statusBarHeight = insets?.top ?? 0;
-		const heightDestination = isMasterDetail ? headerHeight + statusBarHeight : 0;
+		const { theme, currentDepartment, onDepartmentSelected, departments } = this.props;
+		const heightDestination = 0;
 		const translateY = this.animatedValue.interpolate({
 			inputRange: [0, 1],
 			outputRange: [-300, heightDestination] // approximated height of the component when closed/open
 		});
 		const backdropOpacity = this.animatedValue.interpolate({
 			inputRange: [0, 1],
-			outputRange: [0, themes[theme].backdropOpacity]
+			outputRange: [0, themes[theme!].backdropOpacity]
```

Comment:
Here if you don't have the theme as a prop and it comes undefined it will give an error, because the "non-null assertion" doesn't work that way, it will be undefined there the same way.
As it can come undefined the correct thing is to add a fall back if it is undefined.

Reply:
Add a check or confirm that the theme will always come out correct.

Reply:
Unfortunately, this non-null assertion is happening because the HOC `withTheme`, that must have the theme but isn't passing to the component as a prop outside the interface. So because of this, we need to insert a theme in the interface, but as optional just to make TS happy when calling the component.

Reply:
ok, good point, thanks for explaining.